examples/librimix/librimix.sh

```diff
@@ -37,4 +39,27 @@ lhotse cut mix-by-recording-id librimix/cuts_sources.yml.gz librimix/cuts_mix_dy
 # Prepare cuts with feature-domain mixes performed on-the-fly - noisy
 lhotse cut mix-by-recording-id librimix/cuts_sources.yml.gz librimix/cuts_noise.yml.gz librimix/cuts_mix_dynamic_noisy.yml.gz
 
+# The next step is truncation - it makes sure that the cuts all have the same duration and makes them easily batchable
+
+# Truncate the pre-mixed cuts
+lhotse cut truncate \
+  --max-duration $DURATION \
+  --offset-type random \
+  --preserve-id \
+  librimix/cuts_mix.yml.gz librimix/cuts_mix_${DURATION}s.yml.gz
+
+# Truncate the dynamically-mixed clean cuts
```

Comment:
what do you mean by "clean cuts" if they are mixed or noisy?

Reply:
clean => non-noisy

Reply:
Oh, I just noticed that it was a typo